packages/sampler/stories/default/About.js

```diff
@@ -110,9 +110,11 @@ export const ATourOfSampler = () => (
 			style={{bottom: riSafe(edgeDotKeepout), left: 114}}
 			length={30}
 		>
-			<b>Knobs</b> tab lets you adjust component properties
+			<b>Controls</b> tab lets you adjust component properties
 		</HintDialog>
 	</div>
 );
 
+boolean('text centered', ATourOfSampler);
```

Comment:
Shouldn't add `BodyText` as the last parameter?

Reply:
I think we don't need to add it. The 3rd parameter (which is the last parameter here) is passed to the config object. 
Then, the config object is used to determine the default props and group id (which is the category id in addon-controls).
However, in this case, it doesn't use default props and group id.

Reply:
🆗